tests/encode_images.rs

```diff
@@ -260,4 +260,35 @@ fn test_signed() {
         assert_eq!(decoder.get_tag(Tag::Unknown(65030)).unwrap().into_i32_vec().unwrap(), [-1, 100]);
         assert_eq!(decoder.get_tag(Tag::Unknown(65031)).unwrap().into_i32_vec().unwrap(), [-1_i32, 100, 2, 100]);
     }
-}
\ No newline at end of file
+}
+
+#[test]
+/// check multipage image handling
+fn test_multipage_image() {
+    let mut img_file = tempfile::tempfile().unwrap();
```

Comment:
This should not need file io.

```suggestion
    let mut img_file = tempfile::tempfile().unwrap();
```

Reply:
But other examples already use `tempfile` so it doesn't matter greatly.